src/libs/PolicyUtils.ts

```diff
@@ -0,0 +1,221 @@
+import {OnyxCollection, OnyxEntry} from 'react-native-onyx';
+import Str from 'expensify-common/lib/str';
+import CONST from '../CONST';
+import ONYXKEYS from '../ONYXKEYS';
+import {PersonalDetails, Policy, PolicyMembers, PolicyTags} from '../types/onyx';
+
+type MemberEmailsToAccountIDs = Record<string, string>;
+type PersonalDetailsList = Record<string, PersonalDetails>;
+type UnitRate = {rate: number};
+
+/**
+ * Filter out the active policies, which will exclude policies with pending deletion
+ * These are policies that we can use to create reports with in NewDot.
+ */
+function getActivePolicies(policies: OnyxCollection<Policy>): Policy[] | undefined {
+    if (!policies) {
+        return;
+    }
+    return (Object.values(policies) ?? []).filter<Policy>(
+        (policy): policy is Policy =>
+            policy !== null && policy && (policy.isPolicyExpenseChatEnabled || policy.areChatRoomsEnabled) && policy.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE,
+    );
+}
```

Comment:
```suggestion
function getActivePolicies(policies: OnyxCollection<Policy>): Policy[] {
    return Object.values(policies ?? {}).filter(
        (policy): policy is Policy => !!policy && (policy.isPolicyExpenseChatEnabled || policy.areChatRoomsEnabled) && policy.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE,
    );
}
```

We can simplify a little bit here, and `Object.values(policies ?? {})` is safer because won't crash if `policies` is `null/undefined`